src/app/components/Card/Card.tsx

```diff
@@ -28,27 +28,25 @@ export default function Card({ diary, deleteCard }: CardProps): JSX.Element {
 const CardContainer = styled.article`
   position: relative;
   border-radius: 0.5rem;
-  border: 3px solid #5adeff;
+  border: 3px solid var(--card-bg-dark);
```

Comment:
there :D color bg dark is not used for background :D 
maybe keep this in mind for next time ;)